### ISSUE TITLE
accent-color CSS property

### ISSUE THREAD

**Issue by josepharhar** [mozilla/standards-positions#540]
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: accent-color CSS property
* Specification or proposal URL: https://drafts.csswg.org/css-ui-4/#widget-accent
* Caniuse.com URL (optional): 
* Bugzilla URL (optional): https://bugzilla.mozilla.org/show_bug.cgi?id=1705605
* Mozillians who can provide input (optional): @emilio

### Other information

Here are some threads of active discussion:
- https://groups.google.com/a/chromium.org/g/blink-dev/c/WwYkLjbGhoA
- https://github.com/w3c/csswg-drafts/issues/6159

**Comment by annevk** [comment]
Indeed. I suggest we close this without dashboard entry as worth prototyping.
